core/templates/dev/head/pages/collection-editor-page/collection-editor-page.constants.ts

```diff
@@ -27,9 +27,6 @@ export class CollectionEditorPageConstants {
   public static COLLECTION_TITLE_INPUT_FOCUS_LABEL =
     'collectionTitleInputFocusLabel';
 
-  public static SEARCH_EXPLORATION_URL_TEMPLATE =
```

Comment:
Remove it from the *ajs.ts file as well.

Reply:
Done